app/models/manageiq/providers/azure/cloud_manager/refresh_parser.rb

```diff
@@ -141,41 +116,6 @@ def get_images
         process_collection(images, :vms) { |image| parse_image(image) }
       end
 
-      def get_security_groups
-        security_groups = gather_data_for_this_region(@nsg)
-        process_collection(security_groups, :security_groups) { |sg| parse_security_group(sg) }
-      end
-
-      def get_vm_security_groups(instance)
-        get_vm_nics(instance).collect do |nic|
-          sec_id = nic.properties.try(:network_security_group).try(:id)
-          @data_index.fetch_path(:security_groups, sec_id) if sec_id
-        end.compact
-      end
-
-      def get_vm_nics(instance)
-        nic_ids = instance.properties.network_profile.network_interfaces.collect(&:id)
-        @network_interfaces.find_all { |nic| nic_ids.include?(nic.id) }
-      end
-
-      def process_collection(collection, key)
-        @data[key] ||= []
-
-        return if collection.nil?
-
-        collection.each do |item|
-          uid, new_result = yield(item)
-          @data[key] << new_result
-          @data_index.store_path(key, uid, new_result)
-        end
-      end
-
-      def gather_data_for_this_region(arm_service, method = "list")
-        @data[:resource_groups].collect do |resource_group|
-          arm_service.send(method, resource_group[:name])
-        end.flatten
-      end
```

Comment:
@Ladas 
Why are the 2 methods above being moved as part of this PR? They are not related to networking.